main.py

```diff
@@ -3,8 +3,10 @@
 
 def main():
     result = requests.get('https://svt.se')
-    print(result.text)
+    title_part1 = result.text.split('<title data-react-helmet="true">')[1]
+    title = title_part1.split('</title>')[0]
+    print(title)
```

Comment:
Maybe you should have something like 

print('title:', title)

so the user understands what it is you are printing